<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Duhas-Air" failures="1" tests="1" name="Default test" time="109.315" errors="0" timestamp="2021-09-10T17:21:44 EDT">
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="108.898">
    <failure type="com.twilio.exception.ApiException" message="Authenticate">
      <![CDATA[com.twilio.exception.ApiException: Authenticate
at com.twilio.rest.api.v2010.account.MessageReader.pageForRequest(MessageReader.java:243)
at com.twilio.rest.api.v2010.account.MessageReader.firstPage(MessageReader.java:170)
at com.twilio.rest.api.v2010.account.MessageReader.read(MessageReader.java:150)
at com.twilio.base.Reader.read(Reader.java:24)
at pageObjectModel.ShareProject_POM.getMessages2(ShareProject_POM.java:167)
at pageObjectModel.ShareProject_POM.getMessage2(ShareProject_POM.java:162)
at pageObjectModel.ShareProject_POM.fetchOtp(ShareProject_POM.java:173)
at stepDefinition.ShareaProject.And_otp_is_fetched(ShareaProject.java:77)
at âœ½.otp is fetched(file:///Users/duhachowdhury/eclipse-workspace/AlteonIo/src/test/java/Features/ShareProject.feature:16)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- Default test -->
